index.js

```diff
@@ -56,17 +47,17 @@ app.post('/admin_poll', (req, res) => {
 
   adminPolls[id] = req.body.adminPoll; adminVotes[id] = adminTally
   adminPolls[`${id}`]['refId'] = id; liveAdminPolls[liveId] = adminPolls[id]
```

Comment:
Will use just the id next. Also will go line by line instead.

Where would I extract this logic? I need this to happen in the post request. Or do you mean make functions that I then call in the post? That would make sense.